src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderTransactionHistoryResourceTest.java

```diff
@@ -232,8 +232,8 @@ public void When_GetOffenderTransactionHistory_And_BadFromDateFormat_Then_ErrorR
             OFFENDER_NO);
 
         assertThat(response.getBody().getStatus().intValue()).isEqualTo(HTTP_BAD_REQ);
-        assertThat(response.getBody().getDeveloperMessage()).startsWith("Unable to parse date time value \"2019-30-17\"");
-        assertThat(response.getBody().getUserMessage()).startsWith("Unable to parse date time value \"2019-30-17\"");
+        assertThat(response.getBody().getDeveloperMessage()).startsWith("Invalid value for MonthOfYear (valid values 1 - 12): 30");
```

Comment:
this is a revert to 5.3.4 spring behaviour - bug was introduced in spring 5.3.5 and fixed in spring 5.3.6